src/sentry/static/sentry/app/views/settings/components/forms/richListField.jsx

```diff
@@ -0,0 +1,229 @@
+import _ from 'lodash';
+import PropTypes from 'prop-types';
+import React from 'react';
+import styled from 'react-emotion';
+
+import {t} from 'app/locale';
+import DropdownAutoComplete from 'app/components/dropdownAutoComplete';
+import DropdownButton from 'app/components/dropdownButton';
+import InputField from 'app/views/settings/components/forms/inputField';
+import InlineSvg from 'app/components/inlineSvg';
+import Confirm from 'app/components/confirm';
+
+const ItemList = styled('div')`
+  display: flex;
+  flex-wrap: wrap;
+`;
+
+const Item = styled('span')`
+  display: inline-block;
+  background-color: ${p => p.theme.button.default.background};
+  border: 1px solid ${p => p.theme.button.default.border};
+  border-radius: ${p => p.theme.button.borderRadius};
+  color: ${p => p.theme.button.default.color};
+  cursor: default;
+  font-size: ${p => p.theme.fontSizeSmall};
+  font-weight: 600;
+  line-height: 1;
+  padding: 0;
+  text-transform: none;
+  margin-right: 10px;
+  margin-bottom: 5px;
+`;
+
+const ItemLabel = styled('span')`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding: 8px 12px;
+  white-space: nowrap;
+`;
+
+const ItemIcon = styled('span')`
```

Comment:
You might want to use a styled(Button) here. You'll get accessibility and pointer events for free, and the `borderless size="zero"` mode might get you close to what you have here.

Reply:
I played around with this and wasn't able to make it work (without assuming too much about the internal implementation of Button). Suggestions welcome!

Reply:
I tried to get our current button component to fit and it was pretty clunky to do with styled components. I could add a new `micro` button size though that gets you the styling you're looking for if that would help.

Reply:
Sounds good, thanks! 👍